governance/polls/DAI Direct Deposit Module (D3M) Debt Ceiling Targeting - December 13, 2021.md

```diff
@@ -0,0 +1,52 @@
+---
+title: DAI Direct Deposit Module (D3M) Debt Ceiling Targeting - December 13, 2021
+summary: Rank your preferred options to recommend a target Debt Ceiling for the DAI Direct Deposit Module (D3M).
+discussion_link: https://forum.makerdao.com/t/signal-request-d3m-parameters-guidance-for-momc/11898
+vote_type: Ranked Choice IRV
+categories:
+   - Rates
+   - Risk Variable
+options:
+   0: Abstain
+   1: 25% of Real AAVEv2 DAI Supply
+   2: 30% of Real AAVEv2 DAI Supply
+   3: Neither Indicated Percentage of Real AAVEv2 DAI Supply. 
+start_date: 2021-12-13T16:00:00
+end_date: 2021-12-16T16:00:00
+---
+# Poll: DAI Direct Deposit Module (D3M) Debt Ceiling Targeting - December 13, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [Maker Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, December 13 at 16:00 UTC.
+
+**This is an instant-runoff vote.** 
+- **You may rank any number of options that you would support in an executive vote.** 
+- **Unranked options signal that you would not support the option(s) in an executive vote.**
+- **If you have no preference to any of the listed options, you should vote 'Abstain' as your first and only choice.**
+
+## Review
+
+The community can vote in this poll to rank their support to recommend the following target [Debt Ceilings](https://makerdao.world/en/learn/governance/param-debt-ceiling) for the [DAI Direct Deposit Module (D3M)](https://mips.makerdao.com/mips/details/MIP50):
```

Comment:
Template says MKR holders can vote, not community can vote.

Reply:
Looks like our Generic templates need to be updated :)